### DISCUSSION TITLE
[general] Schedulers recommended for target pipelines and more

### DISCUSSION THREAD

**Discussion by asadow** [ropensci/targets#1243] [General]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

I was curious which scheduler folks recommend for repeating their pipelines on a schedule? Would it be cronR? docker + cron? Temporal?

**Comment by noamross** [comment]
This isn't quite the same thing but we have used GitHub Actions to schedule targets pipelines on our machines with a local runner.

**Comment by asadow** [comment]
I didn't know you could do that. Is it something you would also use/recommend for non-targets pipelines (like a regular ETL job)?

**Comment by noamross** [comment]
We do it all the time, usually running on GitHub Actions runners, only using a local runner when we have some big model-fitting job that requires more time/resources than the GH Actions runners have.

**Comment by asadow** [comment]
That sounds much better than using cron. I'm only familiar with GHA through the `usethis` functions for package development. Our private repo's are on Azure DevOps, so I was considering Azure Pipelines. But I'd prefer to use GHA. Would you just recommend the docs for getting started?
